README.md

```diff
@@ -93,17 +93,27 @@ directing the output to).
 ### Output type
 
 Parameter --output-type controls for output format.
-When parameter is not provided in the command line or value is incorrect, `--output-type` is set to `markdown`
+When parameter is not provided in the command line or value is incorrect,
+`--output-type` is set to `markdown`
 This parameter only applies when `--ouput-format` is `txt`
 
 #### `--output-type=markdown`
 
-Program outputs dependency information in markdown format
+Program outputs dependency information in Markdown format
 
 #### `--output-type=json`
 
 Program outputs dependency information in json format
 
+### Application type
+
+Parameter `--application-type` controls the types of licenses that are 
+allowed. Logic is documented in [Notion](https://www.notion.so/datawire/1-Automate-License-Scan-and-Information-Files-31f4cd0f58f645f0afb922cfd710df81) 
+
+This parameter can take two values:
+- Internal: Applies validation for applications that run on customer infrastructure
```

Comment:
Should add ` ` around the values - and make sure they match expected string case